frontend/webpack.config.js

```diff
@@ -148,5 +149,29 @@ module.exports = {
             indent: '\t',
         }),
         new CompressionPlugin(),
+        new FaviconsWebpackPlugin({
+            logo: '../backend/static/img/logo.svg',
+            cache: true,
+            outputPath: './../../backend/static/img/favicon',
+            // Prefix path for generated assets in generated html
+            prefix: 'img/favicon',
+            inject: false,
+            favicons: {
+                appName: 'match4everyone',
```

Comment:
I would suggest putting this configuration inside backend maybe as json and load it here? But that is probably an extra issue and depending on the docs as right now configs are split to multiple places anyway

Reply:
Yes, In the end, one config json would be a good thing, but also somewhat complicated to implement. I would treat it as a separate issue.